src/flowgraph/FlowGraphNode.h

```diff
@@ -40,8 +40,16 @@
 
 // Set this to 1 if using it inside the Android framework.
 // This code is kept here so that it can be moved easily between Oboe and AAudio.
+#ifndef FLOWGRAPH_ANDROID_INTERNAL
 #define FLOWGRAPH_ANDROID_INTERNAL 0
+#endif
 
+// Set this to a name that will prevent AAudio from calling into Oboe.
+#ifndef FLOWGRAPH_OUTER_NAMESPACE
+#define FLOWGRAPH_OUTER_NAMESPACE oboe
```

Comment:
Would it make sense to define the namespace based on whether `FLOWGRAPH_ANDROID_INTERNAL` is set to `0`?

Reply:
I did that originally and then decided this was better.

I want to be able to set the namespace to aaudio::flowgraph when we use it in AAudio.
Also if someone wants to steal it, this will make it easier to customize the namespace.